extensions/jms/src/main/java/org/mule/extensions/jms/internal/connection/provider/activemq/ActiveMQConnectionFactoryProvider.java

```diff
@@ -77,7 +76,7 @@ ConnectionFactory createDefaultConnectionFactory() throws ActiveMQConnectionExce
       String message = format("Failed to create a default Connection Factory for ActiveMQ using the [%s] implementation: ",
                               getFactoryClass());
       LOGGER.error(message, e);
-      throw new ActiveMQConnectionException(createStaticMessage(message), e);
+      throw new ActiveMQException(message, e);
```

Comment:
think on the guy using the connector. He would need to add different error handlers depending on the broker he selected.  That would be a problem with the connectivity service.

Reality is that you will handle the error in the same way regardless of the broker. All errors represent the same exceptional condition. There's no gain in having different error types